### PR TITLE
Ck feature/ck 35 home change

### PR DESCRIPTION
**Pull request by HaiVT26**
Home change

### CODE REVIEW

File: ClearKeep/Modules/Favourite/CKFavouriteViewController.swift [hunk 1]
```diff
@@ -41,15 +41,15 @@ class CKFavouriteViewController: CKRecentListViewController {
...
-    }
+//    override func bindingTheme() {
+//        super.bindingTheme()
+//
+//        // Binding navigation bar color
+//        themeService.attrsStream.subscribe(onNext: { [weak self] (theme) in
+//            self?.defaultBarTintColor = themeService.attrs.primaryBgColor
+//            self?.barTitleColor = themeService.attrs.primaryTextColor
+//        }).disposed(by: disposeBag)
+//    }
```
Comment (HoaPham2711) [inline ClearKeep/Modules/Favourite/CKFavouriteViewController.swift]: Pls remove the unsued code

File: ClearKeep/Modules/Home/CKRoomPageViewController.swift [hunk 2]
```diff
@@ -11,11 +11,11 @@ import UIKit
...
     var missedItemCount: Int = 0
-
-    override func reloadData(rooms: [MXKRecentCellData]) {
+    
+    override func reloadData(rooms: [[MXKRecentCellData]]) {
         super.reloadData(rooms: rooms)
         
         // keep missedItemCount
-        self.missedItemCount = rooms.filter({ $0.roomSummary.membership == MXMembership.invite || $0.hasUnread || $0.notificationCount > 0 }).count
+        self.missedItemCount = rooms[0].filter({ $0.roomSummary.membership == MXMembership.invite || $0.hasUnread || $0.notificationCount > 0 }).count
```
Comment (HoaPham2711) [inline ClearKeep/Modules/Home/CKRoomPageViewController.swift]: Pls make sure rooms have at least one item before getting rooms[0]. The app may be crashed

File: ClearKeep/Modules/Home/CkHomeViewController.swift [hunk 3]
```diff
@@ -10,49 +10,30 @@ import Foundation
...
-        bindingTheme()
+        NotificationCenter.default.addObserver(self, selector: #selector(userInfoDidChanged(_:)), name: NSNotification.Name.mxkAccountUserInfoDidChange, object: nil) 
     }
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        setupNavigationBar()
+        
+        self.setupNavigationBar()
+        self.delegate = self
```
Comment (HoaPham2711) [inline ClearKeep/Modules/Home/CkHomeViewController.swift]: Why do we need to set a delegate in the viewWillAppear function?

File: ClearKeep/Extensions/CKViewController+Extension.swift [hunk 4]
```diff
@@ -26,9 +26,10 @@ extension UIViewController {
     /**
      Show alert in Self
      */
-    func showAlert(_ message: String) {
-        let alert = UIAlertController(title: "ClearKeep", message: message, preferredStyle: UIAlertControllerStyle.alert)
-        alert.addAction(UIAlertAction(title: "OK", style: UIAlertActionStyle.default, handler: nil))
+    func showAlert(_ message: String) { 
+        let error = message == "No known servers" ? Bundle.mxk_localizedString(forKey: "room_error_join_failed_empty_room") : message
```
Comment (HoaPham2711) [inline ClearKeep/Extensions/CKViewController+Extension.swift]: Don't check according to the message content and the show alert is a generic function so you shouldn't do a check in here

File: ClearKeep/Modules/Home/CKRoomPageViewController.swift (hunk 2 above)
Comment (HoaPham2711) [reply ClearKeep/Modules/Home/CKRoomPageViewController.swift]: Need to have a validation before

File: ClearKeep/Modules/Favourite/CKFavouriteViewController.swift (hunk 1 above)
Comment (HaiVT26) [reply ClearKeep/Modules/Favourite/CKFavouriteViewController.swift]: I do it, thank!

File: ClearKeep/Modules/Home/CKRoomPageViewController.swift (hunk 2 above)
Comment (HaiVT26) [reply ClearKeep/Modules/Home/CKRoomPageViewController.swift]: Class CKRoomPageViewController not use anymore/ I'll remove it.

File: ClearKeep/Modules/Home/CkHomeViewController.swift (hunk 3 above)
Comment (HaiVT26) [reply ClearKeep/Modules/Home/CkHomeViewController.swift]: It my mistake, I'll put it in viewDidLoad

File: ClearKeep/Extensions/CKViewController+Extension.swift (hunk 4 above)
Comment (HaiVT26) [reply ClearKeep/Extensions/CKViewController+Extension.swift]: Thank for review. I'll check it in local

**Pull request merged**